src/App.js

```diff
@@ -10,11 +14,29 @@ class App extends React.Component {
 
   render() {
     return (
-      <div>
-        <div className="calculator-body">
-          <Calculator />
-        </div>
-      </div>
+
+      <>
+        <Navbar />
+        <Switch>
+          <div>
+
+            <Route exact path="/">
+              <Home />
+            </Route>
+            <Route path="/calculator">
+              <div className="calculator-body">
+                <p>
+                  {' Lets do some Maths!' }
+                </p>
```

Comment:
[OPTIONAL]

- I would like to give you one suggestion here. Since you are making every page as a component, it's better to put all those div inside the `Calculator` component itself.  So that it all belongs to one component and makes the codebase clearner.